metabase/metabase
# Simplify Slack setup

Fixes https://github.com/metabase/metabase/issues/20050

Removes the manifest step and inlines the manifest into the URL.

How to test:
- Go to Admin > Slack
- Make sure it is possible to setup a Slack app using the steps on this page.
